app/controllers/v0/sign_in_controller.rb

```diff
@@ -311,6 +286,15 @@ def delete_cookies
       cookies.delete(SignIn::Constants::Auth::INFO_COOKIE_NAME)
     end
 
+    def get_user_auth_info(user)
```

Comment:
What I'd do here is make this a default parameter:
`def get_user_auth_info(user= @current_user)`
And then any place that needs to explicitly pass in a user (because `@current_user` hasn't been set up yet), can invoke this with `get_user_auth_info(User.find(user_uuid))`. I think that might simplify this function a bit, and make it so you don't have to introspect on the type of object that `user` is